caom2/caom2/tests/test_obs_reader_writer.py

```diff
@@ -239,31 +240,17 @@ def test_complete_derived(self):
                                       version)
 
     def test_versions(self):
```

Comment:
This method does not have any tests for 25 against other docs. Was there a decision made to not keep this cross-version checking going forward for 25?

Reply:
I've just missed that by mistake. Thanks for pointing it out. Fixed now.